controllers/chat/moveToAnon.js

```diff
@@ -5,6 +5,7 @@ const {responseError, responseSuccess} = require('../../utils/Responses');
 const {CHANNEL_TYPE} = require('../../helpers/constants');
 const UsersFunction = require('../../databases/functions/users');
 const Getstream = require('../../vendor/getstream');
+const BetterSocialConstantListUtils = require('../../services/bettersocial/constantList/utils');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new import statement for `BetterSocialConstantListUtils` is added correctly. It seems to be used later in the code for getting random emoji and color.
```diff
+ const BetterSocialConstantListUtils = require('../../services/bettersocial/constantList/utils');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->